### DISCUSSION TITLE
Quasar QBtnDropdown API

### DISCUSSION THREAD

**Discussion by harshalkumar-ishi** [elimintz/justpy#689] [General]
In Quasar how to QBtnDropdown to use show and hide methods. 
There is no method given in the documentation on how to user this feature.

**Comment by freemant2000** [comment]
![image](https://github.com/justpy-org/justpy/assets/1174319/74d46a20-41d4-42d4-acd3-7f8eacb192d1)
It seems you can just use the model (value) to control showing or hiding of the menu.

**Reply by harshalkumar-ishi** [reply]
okay i will check.

**Reply by harshalkumar-ishi** [reply]
thanks for the reply.
